src/popup/router/components/InviteItem.vue

```diff
@@ -1,36 +1,58 @@
 <template>
   <div class="invite-row">
     <div class="invite-info">
-      <span class="primary">{{ balance }} {{ $t('pages.appVUE.aeid') }}</span>
+      <span class="primary">
+        <span class="balance">{{ balance }}</span>
+      </span>
       <!--eslint-disable-line vue-i18n/no-raw-text-->
-      ({{ (balance * current.currencyRate).toFixed(2) }}
+      ({{ currencySigns[current.currency.toUpperCase()] }}
       <!--eslint-disable-next-line vue-i18n/no-raw-text-->
-      {{ current.currency.toUpperCase() }})
+      {{ (balance * current.currencyRate).toFixed(2) }})
       <span class="date">{{ createdAt | formatDate }}</span>
     </div>
     <div class="invite-link">
       <span>{{ link }}</span>
       <button class="invite-link-copy" v-clipboard:copy="link"><CopyIcon /></button>
     </div>
     <template v-if="!topUp">
-      <Button half dark @click="topUp = true">{{ $t('pages.invite.top-up') }}</Button>
-      <Button half @click="claim">{{ $t('pages.invite.claim') }}</Button>
+      <div class="center">
```

Comment:
Is it uses styles from [Loader](https://github.com/aeternity/superhero-wallet/blob/083616d3321619c8be29c306d9b4f5795e2cc14b/src/popup/router/components/Loader.vue#L85-L89) component? We should avoid the usage of global styles because they complicate the maintenance. I propose just to use flexbox instead.

Reply:
I used the global style .center from the base.scss. I wasn't aware there is a duplicate in Loader component. I am going to change the class name.